src/Apropos/HasPermutationGenerator.hs

```diff
@@ -5,7 +5,8 @@ module Apropos.HasPermutationGenerator (
   abstract,
   gotoSum,
   abstractsProperties,
-  (|:->),
+  (&&&),
+  (>>>),
```

Comment:
This is a nice change. I think Control.Arrow uses these operators.

Reply:
Yeah the names are inspired by/stolen from Control.Arrow